<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="1.094" tests="7" failures="1">
  <testsuite name="Root Suite" timestamp="2024-09-18T19:55:17" tests="0" file="cypress\e2e\services\auditorsList\auditorsList.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Auditors List E2E Tests" timestamp="2024-09-18T19:55:17" tests="7" time="1.083" failures="1">
    <testcase name="Auditors List E2E Tests &quot;before each&quot; hook for &quot;should verify Clerk is loaded&quot;" time="0.000" classname="&quot;before each&quot; hook for &quot;should verify Clerk is loaded&quot;">
      <failure message="`cy.visit()` failed trying to load:

http://localhost:3000/services/auditorsList

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  &gt; Error: Exceeded maxRedirects. Probably stuck in a redirect loop http://localhost:3000/sign-in?redirect_url=http%3A%2F%2Flocalhost%3A3000%2Fsign-in%3Fredirect_url%3Dhttp%253A%252F%252Flocalhost%253A3000%252Fsign-in%253Fredirect_url%253Dhttp%25253A%25252F%25252Flocalhost%25253A3000%25252Fsign-in%25253Fredirect_url%25253Dhttp%2525253A%2525252F%2525252Flocalhost%2525253A3000%2525252Fsign-in%2525253Fredirect_url%2525253Dhttp%252525253A%252525252F%252525252Flocalhost%252525253A3000%252525252Fsign-in%252525253Fredirect_url%252525253Dhttp%25252525253A%25252525252F%25252525252Flocalhost%25252525253A3000%25252525252Fsign-in%25252525253Fredirect_url%25252525253Dhttp%2525252525253A%2525252525252F%2525252525252Flocalhost%2525252525253A3000%2525252525252Fservices%2525252525252FauditorsList

Common situations why this would fail:
  - you don&apos;t have internet access
  - you forgot to run / boot your web server
  - your web server isn&apos;t accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Auditors List E2E Tests`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://localhost:3000/services/auditorsList

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: Exceeded maxRedirects. Probably stuck in a redirect loop http://localhost:3000/sign-in?redirect_url=http%3A%2F%2Flocalhost%3A3000%2Fsign-in%3Fredirect_url%3Dhttp%253A%252F%252Flocalhost%253A3000%252Fsign-in%253Fredirect_url%253Dhttp%25253A%25252F%25252Flocalhost%25253A3000%25252Fsign-in%25253Fredirect_url%25253Dhttp%2525253A%2525252F%2525252Flocalhost%2525253A3000%2525252Fsign-in%2525253Fredirect_url%2525253Dhttp%252525253A%252525252F%252525252Flocalhost%252525253A3000%252525252Fsign-in%252525253Fredirect_url%252525253Dhttp%25252525253A%25252525252F%25252525252Flocalhost%25252525253A3000%25252525252Fsign-in%25252525253Fredirect_url%25252525253Dhttp%2525252525253A%2525252525252F%2525252525252Flocalhost%2525252525253A3000%2525252525252Fservices%2525252525252FauditorsList

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Auditors List E2E Tests`
    at <unknown> (http://localhost:3000/__cypress/runner/cypress_runner.js:131955:74)
    at visitFailedByErr (http://localhost:3000/__cypress/runner/cypress_runner.js:131354:12)
    at <unknown> (http://localhost:3000/__cypress/runner/cypress_runner.js:131954:11)
    at tryCatcher (http://localhost:3000/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:3000/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:3000/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:3000/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:3000/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:3000/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:3000/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:3000/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.eval (webpack://leonidbaichi/./cypress/e2e/services/auditorsList/auditorsList.cy.js:20:7)

From Node.js Internals:
  Error: Exceeded maxRedirects. Probably stuck in a redirect loop http://localhost:3000/sign-in?redirect_url=http%3A%2F%2Flocalhost%3A3000%2Fsign-in%3Fredirect_url%3Dhttp%253A%252F%252Flocalhost%253A3000%252Fsign-in%253Fredirect_url%253Dhttp%25253A%25252F%25252Flocalhost%25253A3000%25252Fsign-in%25253Fredirect_url%25253Dhttp%2525253A%2525252F%2525252Flocalhost%2525253A3000%2525252Fsign-in%2525253Fredirect_url%2525253Dhttp%252525253A%252525252F%252525252Flocalhost%252525253A3000%252525252Fsign-in%252525253Fredirect_url%252525253Dhttp%25252525253A%25252525252F%25252525252Flocalhost%25252525253A3000%25252525252Fsign-in%25252525253Fredirect_url%25252525253Dhttp%2525252525253A%2525252525252F%2525252525252Flocalhost%2525252525253A3000%2525252525252Fservices%2525252525252FauditorsList
      at r (<embedded>:445:85399)
      at tryCatcher (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\util.js:16:23)
      at Promise._settlePromiseFromHandler (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\promise.js:512:31)
      at Promise._settlePromise (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\promise.js:569:18)
      at Promise._settlePromise0 (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\promise.js:614:10)
      at Promise._settlePromises (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\promise.js:694:18)
      at _drainQueueStep (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\async.js:138:12)
      at _drainQueue (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\async.js:131:9)
      at Async._drainQueues (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\async.js:147:5)
      at Immediate._onImmediate (C:\Users\DELL\AppData\Local\Cypress\Cache\13.14.2\Cypress\resources\app\node_modules\bluebird\js\release\async.js:17:14)
      at process.processImmediate (node:internal/timers:476:21)]]></failure>
    </testcase>
  </testsuite>
</testsuites>